ckcp/openshift_dev_setup.sh

```diff
@@ -275,9 +275,17 @@ patches:
         value: $ckcp_route " >>"$ckcp_temp_dir/kustomization.yaml"
 
   echo -n "  - kcp $kcp_version: "
-  kubectl apply -k "$ckcp_temp_dir" >/dev/null 2>&1
+  kubectl apply -k "$ckcp_temp_dir" >/dev/null
   # Check if ckcp pod status is Ready
-  kubectl wait --for=condition=Ready -n $ns pod -l=app=kcp-in-a-pod --timeout=90s >/dev/null
+  kubectl wait --for=condition=Ready -n $ns pod -l=app=kcp-in-a-pod --timeout=90s >/dev/null 2>&1 || debug="true"
+  if [ "$debug" == "true" ]; then
+    printf "Debug: print out pod state\n"
+    kubectl -n $ns get pod
```

Comment:
please use describe to get pods related status and events
`kubectl -n $ns describe pods

Reply:
fixed